Tools/TestWebKitAPI/Tests/WebKitCocoa/EventAttribution.mm

```diff
@@ -524,36 +524,28 @@ static void attemptConnectionInProcessWithoutEntitlement()
 }
 
 #if PLATFORM(MAC)
-static void setInjectedBundleClient(WKWebView *webView, Vector<String>& consoleMessages)
-{
-    WKPageInjectedBundleClientV0 injectedBundleClient = {
-        { 0, &consoleMessages },
-        [] (WKPageRef, WKStringRef messageName, WKTypeRef message, const void* clientInfo) {
-            auto& consoleMessages = *reinterpret_cast<Vector<String>*>(const_cast<void*>(clientInfo));
-            if (WKStringIsEqualToUTF8CString(messageName, "ConsoleMessage"))
-                consoleMessages.append(Util::toNS((WKStringRef)message));
-        },
-        nullptr,
-    };
-    WKPageSetPageInjectedBundleClient(webView._pageForTesting, &injectedBundleClient.base);
-};
-
-static RetainPtr<TestWKWebView> webViewWithOpenInspector(WKWebViewConfiguration *configuration)
+static RetainPtr<TestWKWebView> webViewWithOpenInspector(WKWebViewConfiguration *configuration, id<WKUIDelegate> uiDelegate)
```

Comment:
It looks like we're not consistent about this in existing code, but it seems like in the majority of cases we insert a space between `id` and `<`.

Reply:
There's a WebKit style checker enforcing that we don't add the space:

Tools/TestWebKitAPI/Tests/WebKitCocoa/EventAttribution.mm:527: error: [spacing/objc-protocol] Protocol names shouldn't have a space before them.